content/sql/concepts/dates/terms/dateadd/dateadd.md

```diff
@@ -0,0 +1,101 @@
+---
+Title: 'DATEADD()'
+Description: 'Adds a time or date interval to a specified date and returns it. Versions are available in SQL Server and MySQL.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'SQL Server'
+  - 'MySQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+The **`DATEADD()`** function in SQL Server and `DATE_ADD()` in MySQL adds a time or date interval to a specified date and returns it.
+
+## SQL Server Syntax
+
+The `DATEADD()` function in SQL Server has three parameters:
+
+```pseudo
+DATEADD(interval, amount, date)
+```
+
+- `date` is the date being added to.
+- `amount` is the amount of `interval` to add to `date`. A negative amount subtracts the interval from the date.
+- `interval` is the date or time interval to add to `date`. It can be one of the following formats:
+  - Year: year, yyyy, yy
+  - Quarter: quarter, qq, q
+  - Week: week, ww, wk
+  - Weekday: weekday, dw, w
+  - Second: second, ss, s
+  - Month: month, mm, m
+  - Minute: minute, mi, n
+  - Millisecond: millisecond, ms
+  - Hour: hour, hh
+  - Day of Year: dayofyear
+  - Day: day, dy, y
```

Comment:
```suggestion
  - Year: `year`, `yyyy`, `yy`
  - Quarter: `quarter`, `qq`, `q`
  - Week: `week`, `ww`, `wk`
  - Weekday: `weekday`, `dw, `w`
  - Second: `second`, `ss`, `s`
  - Month: `month`, `mm`, `m`
  - Minute: `minute`, `mi`, `n`
  - Millisecond: `millisecond`, `ms`
  - Hour: `hour`, `hh`
  - Day of Year: `dayofyear`
  - Day: `day`, `dy`, `y`
```

Since these shorthands are what's actually used in the method, I suggest surround in back-ticks.